### PR TITLE
Convert noteblocks for mozilla/add-ons/webextensions folder

### PR DESCRIPTION
**Pull request by queengooborg**
This PR converts the noteblocks for the 'mozilla/add-ons/webextensions' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 7.

### CODE REVIEW

File: files/en-us/mozilla/add-ons/webextensions/api/types/index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Defines the `BrowserSetting` type, which is used to represent a browser setting.
 
 {{WebExtExamples("h2")}}
 
-> **Note:**
+> [!NOTE]
 >
```
Comment (Josh-Cena) [inline files/en-us/mozilla/add-ons/webextensions/api/types/index.md:21]:
```suggestion
> [!NOTE]
```

File: files/en-us/mozilla/add-ons/webextensions/work_with_the_cookies_api/index.md [hunk 2]
```diff
@@ -88,7 +89,8 @@ Firefox provides three types of cookie store:
 - The default store, which stores cookies from normal browsing.
 - Private browsing mode stores, which stores cookies created during a private browsing session. These stores and any cookies they contain are removed when the related private browsing window closes.
 
-  > **Note:**
+  > [!NOTE]
+  >
```
Comment (Josh-Cena) [inline files/en-us/mozilla/add-ons/webextensions/work_with_the_cookies_api/index.md:93]:
```suggestion
  > [!NOTE]
```

File: files/en-us/mozilla/add-ons/webextensions/working_with_the_tabs_api/index.md [hunk 3]
```diff
@@ -76,7 +77,9 @@ To see how {{WebExtAPIRef("tabs.query()")}} and {{WebExtAPIRef("tabs.Tab")}} are
     }
     ```
 
-    > **Note:**
+    > [!NOTE]
+    >
+    > >
```
Comment (Josh-Cena) [inline files/en-us/mozilla/add-ons/webextensions/working_with_the_tabs_api/index.md:82]:
```suggestion
    > [!NOTE]
```

**Review by Josh-Cena — APPROVED** [review]

File: files/en-us/mozilla/add-ons/webextensions/working_with_the_tabs_api/index.md (hunk 3 above)
Comment (queengooborg) [reply files/en-us/mozilla/add-ons/webextensions/working_with_the_tabs_api/index.md]: Okay, now I _know_ I searched for all these cases and fixed them before submitting these PRs...  Something strange is going on within the Husky precommit hooks, and I don't know what.  :/

Thanks for catching all these!

**Pull request merged**